Add Named_Logos

zach-morris opened the pull request.

zach-morris committed:
Add Named_Logos

marrikhong commented:
[Named_Logos/._Akira (Europe).png]
[Named_Logos/._Nigel Mansell's World Championship (Europe).png
[Named_Logos/._Skeleton Krew (Europe).png
[Named_Logos/._Syndicate (Europe) (En,Fr,It).png]

4 hidden (unneeded?) files

zach-morris commented:
Sorry i missed the comment. Mac for whatever reason decides to put these hidden files all over the place.  I've updated the gitignore to remove them (I had added it to most but looks like i missed this one)

marrikhong commented:
No problemo. Thanks for the upload!
Please note that some files are 'symbolic links' and not real files and can't be displayed (at least under Windows and on the web Github app, for example : Chaos Engine, The (Europe) (Spectacular Voyage).png). I haven't checked in Retroarch yet. 
Cheers!
